localization/languages/pt-PT.json

```diff
@@ -104,22 +104,22 @@
         "settingsPreferencesHeading": "Preferências",
         "settingsRestartRequired": "Tem que reiniciar o navegador para aplicar as alterações.",
         "settingsPrivacyHeading": "Bloqueio de conteúdo",
-        "settingsContentBlockingLevel0": null, //missing translation
-        "settingsContentBlockingLevel1": null, //missing translation
-        "settingsContentBlockingLevel2": null, //missing translation
-        "settingsContentBlockingExceptions": null, //missing translation
+        "settingsContentBlockingLevel0": "Permitir todos os anúncios e 'trackers'",
+        "settingsContentBlockingLevel1": "Bloquear anúncios e trackers de terceiros",
+        "settingsContentBlockingLevel2": "Bloquear todos os anúncios e 'trackers'",
+        "settingsContentBlockingExceptions": "Permitir anúncios nestes sites:",
         "settingsBlockScriptsToggle": "Bloquear scripts",
         "settingsBlockImagesToggle": "Bloquear imagens",
         "settingsEasyListCredit": {
-            "unsafeHTML": null //missing translation
+            "unsafeHTML": "Baseado em EasyList e EasyPrivacy, criadas por <a target=\"_blank\" href=\"https://easylist.to/\">The EasyList authors</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">view license</a>)."
```

Comment:
Could you translate the "view license" part here as well?

Reply:
I can, please merge this one. I'll make another one after.

Reply:
OK, sure.